website/src/pages/mpox/single-variant.astro

```diff
@@ -38,19 +33,31 @@ const initialMutations = extractMutations(variantFilter);
                 <div class='bg-blue-50 px-2 py-4'>
                     <div class='mb-2 font-semibold'>Filter dataset:</div>
                     <LocationTimeFilter
-                        fields={ServerSide.mpoxView1.locationFields}
+                        fields={ServerSide.mpoxAnalyzeSingleVariantView.locationFields}
                         initialLocation={baselineFilter}
                         initialDateRange={routeData.baselineFilter.dateRange}
-                        earliestDate={ServerSide.mpoxView1.earliestDate}
-                        customDateRangeOptions={ServerSide.mpoxView1.customDateRangeOptions}
+                        earliestDate={ServerSide.mpoxAnalyzeSingleVariantView.earliestDate}
+                        customDateRangeOptions={ServerSide.mpoxAnalyzeSingleVariantView.customDateRangeOptions}
                     />
                 </div>
                 <div class='border-t-2 bg-green-50 px-2 py-4'>
                     <div class='mb-2 font-semibold'>Search variant:</div>
                     <VariantFilter
                         fields={[
-                            { name: 'clade', label: 'Clade', initialValue: variantFilter.clade, type: 'text' },
-                            { name: 'lineage', label: 'Lineage', initialValue: variantFilter.lineage, type: 'text' },
+                            {
+                                name: ServerSide.mpoxAnalyzeSingleVariantView.cladeField,
+                                label: 'Clade',
+                                initialValue: variantFilter.clade,
+                                type: 'text',
+                                rename: 'clade',
+                            },
+                            {
+                                name: ServerSide.mpoxAnalyzeSingleVariantView.lineageField,
+                                label: 'Lineage',
+                                initialValue: variantFilter.lineage,
+                                type: 'text',
+                                rename: 'lineage',
+                            },
```

Comment:
`clade` and `lineage` are here, because they have to match the keys used in `getLapisVariantQuery`? Let's connect them via common constants?